scripts/index.js

```diff
@@ -37,8 +37,8 @@ config.output.filename = `js/bundle.js`;
 config.output.chunkFilename = `js/[name].chunk.js`;
 
 // update media path destination
-config.module.rules[1].oneOf[0].options.name = `media/[name].[hash:8].[ext]`;
-config.module.rules[1].oneOf[3].options.name = `media/[name].[hash:8].[ext]`;
+config.module.rules[3].oneOf[0].options.name = `media/[name].[hash:8].[ext]`;
```

Comment:
The problem with that approach of just modifying the indexes for the latest version is that we break the retro-compatibility with the `react-scripts < 2.0.3`. I'd say it would be better to find a way to test the `react-scripts` version being used and adapt the indexes accordingly. This way we avoid a breaking change and having to support 2 different versions.

Reply:
If you need help on the matter, feel free to ping me.